.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+## Description
+
+<!--
+Please include a summary of your proposed changes and which issue is being addressed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+-->
+
+## Issue Number
+
+<!--
+Internal Users - Please include the related internal tracking number (e.g. BG-000000).
+External Users - Please link to any relevant github issues as necessary.
+ -->
+
+## Type of change
+
+<!-- Please delete options that are not relevant. -->
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+# How Has This Been Tested?
+
+<!--
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.
+-->
+
+# Checklist:
+
```

Comment:
I know this is in the comments but should we add a box where it says something along the line such as `The ticket or github issue was included in the commit message as a reference`?

Reply:
That's a good idea. The CI should fail it without, but let's add that here too for visibility.